BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.kt

```diff
@@ -466,4 +510,31 @@ open class BraintreeClient @VisibleForTesting internal constructor(
     open fun launchesBrowserSwitchAsNewTask(launchesBrowserSwitchAsNewTask: Boolean) {
         this.launchesBrowserSwitchAsNewTask = launchesBrowserSwitchAsNewTask
     }
+
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    fun getGraphQLMutationName(jsonString: String): String {
+        val regex = """"query"\s*:\s*"(.*?)"""".toRegex()
+        val matchResult = regex.find(jsonString)
+
+        return if (matchResult != null) {
```

Comment:
q: Can we get the value from the GraphQL POST body instead since it's already in JSON?

Reply:
Actually, the body is of type `string`, any ideas? do you have any references I could use?

Reply:
There may be some refactoring we can do to access the JSON directly. We can also clean this up in a separate PR.